smug/importers/twinl_importer.py

```diff
@@ -0,0 +1,58 @@
+from argparse import ArgumentParser
+
+import pandas as pd
+import pkg_resources
+import gzip
+import json
+
+from smug.connection_manager import ConnectionManager
+
+
+class GzImporter:
+    def __init__(self, connection_mananger):
+        self.connection_manager = connection_mananger
+
+    def process_files(self, files):
+        for file in files:
+            self.process_file(file)
+
+    def process_file(self, file):
+        filename = pkg_resources.resource_filename('resources', file)
+
+        with gzip.open(filename, 'rt') as f:
+            print('Sending {}'.format(file))
+            contents = f.read()
+
+            contents = contents.replace('\n', ',')
```

Comment:
for line in f[.readlines()]: otherwise the entire file needs to be loaded into memory.